src/components/header-component/Registration.vue

```diff
@@ -5,6 +5,9 @@
       <input id="surname" v-model="surname" type="text" name="surname" placeholder="Фамилия" />
       <PhoneInput @onchange="onChangePhone" id="phone" placeholder="Телефон" />
       <input id="password" v-model="password" type="password" name="password" placeholder="Пароль" />
+      <div v-if="loading && !additionalFields" class="registration__loader-wrapper">
+        <Loader className="registration__loader" />
```

Comment:
Do you have styles for this classname?

Reply:
@Xoxol no. For now, if I don't send className - it will set default. In this case ,I do not need the default styles, so I just set any name. Can you advise another solution?